lib/std/os/windows/bits.zig

```diff
@@ -1573,3 +1573,21 @@ pub const SYMLINK_FLAG_RELATIVE: ULONG = 0x1;
 
 pub const SYMBOLIC_LINK_FLAG_DIRECTORY: DWORD = 0x1;
 pub const SYMBOLIC_LINK_FLAG_ALLOW_UNPRIVILEGED_CREATE: DWORD = 0x2;
+
+pub const MOUNTMGR_MOUNT_POINT = extern struct {
+    SymbolicLinkNameOffset: ULONG,
+    SymbolicLinkNameLength: USHORT,
+    Reserved1: USHORT,
+    UniqueIdOffset: ULONG,
+    UniqueIdLength: USHORT,
+    Reserved2: USHORT,
+    DeviceNameOffset: ULONG,
+    DeviceNameLength: USHORT,
+    Reserved3: USHORT,
+};
+pub const MOUNTMGR_MOUNT_POINTS = extern struct {
+    Size: ULONG,
+    NumberOfMountPoints: ULONG,
+    MountPoints: [1]MOUNTMGR_MOUNT_POINT,
```

Comment:
I'm not sure about having these VLA elements in our definitions.

Reply:
Hmm, TBH I've taken them verbatim from the NT header files. What would you have in mind instead? An explicit pointer to the buffer?

Reply:
I think in other places we've just been leaving out the VLA element.

Reply:
I had a quick scan through our definitions and it seems we have been including VLAs before, e.g., [here](https://github.com/ziglang/zig/blob/master/lib/std/os/windows/bits.zig#L260) and [here](https://github.com/ziglang/zig/blob/master/lib/std/os/windows/bits.zig#L401). I'm not defending the use of VLAs here, rather I'm suggesting to refactor this out entirely in a subsequent PR. What do you think?

Reply:
> I'm suggesting to refactor this out entirely in a subsequent PR. What do you think?

Sounds fair to me.
Could you open an issue for it?

Reply:
FYI, I've filed the issue -> #6008